src/styles/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  position: relative;
+  max-width: $cell-size * 8 + 6 * $cell-gap;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 40px auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.